### DISCUSSION TITLE
I can't upload the proof of my academic status.

### DISCUSSION THREAD

**Discussion by nahuarce12** [community/community#114653] [GitHub Education]
### Hi everyone,

Hi everyone, 
I'm encountering an issue when attempting to upload a proof of my academic status. The upload process seems to continuously load without providing me with any option to actually upload the proof. Any assistance would be greatly appreciated. Thank you!

![Sin título](https://github.com/community/community/assets/164960010/ac3b7c9b-7fdc-4d33-85b2-c81ad93637b9)

**Comment by mxt1022** [comment]
Try uploading from your cell phone, or switching to a different network
